v1.5 final release: quick hits (#3249)
fishtown-analytics/docs.getdbt.com: website/docs/reference/commands/parse.md

id: "parse"
---

> ## Overview

Reviewer: we're reducing (or eliminating) our use of 'overview' as the intro header as the user will see the title of the page, followed by  the h2.
![Screenshot 2023-04-24 at 16 01 33](https://user-images.githubusercontent.com/89008547/234036708-fae8a114-9dc1-4e78-bc9c-c404cb3308f7.png)

```suggestion
```

The `dbt parse` command parses and validates the contents of your dbt project. If your project contains Jinja or YAML syntax errors, the command will fail.

> It will also produce an artifact with detailed timing information, which is useful to understand parsing times for large projects. For more details, [read about parsing](parsing).

Reviewer: ```suggestion
It will also produce an artifact with detailed timing information, which is useful to understand parsing times for large projects. Refer to [Project parsing](/reference/parsing) for more info.
```

<VersionBlock firstVersion="1.5">

> Starting in v1.5, `dbt parse` will write or return a [manifest](manifest-json), enabling you to introspect dbt's understanding of all the resources in your project.

Reviewer: ```suggestion
Starting in v1.5, `dbt parse` will write or return a [manifest](/reference/artifacts/manifest-json), enabling you to introspect dbt's understanding of all the resources in your project.
```
